orm/utils.go

```diff
@@ -181,18 +181,36 @@ func ToInt64(value interface{}) (d int64) {
 	return
 }
 
-// snake string, XxYy to xx_yy
+// snake string, XxYy to xx_yy , XxYY to xx_yy
```

Comment:
@ysqi yes. Here is the exmaple: `GComment` need to `g_comment`, but now it turn to `gc_omment`